docs/source/conf.py

```diff
@@ -143,7 +143,7 @@
 
 
 favicons = [
-    {"href": "images/logo-dff.svg"},
+    {"href": "images/logo-chatsky.svg"},
```

Comment:
I didn't include this change in #368 because the logo file is old and is a logo for dff. Chatsky has a new logo which will be added a bit later.